<?xml version="1.0" encoding="utf-8" ?>
<launch>
    <arg name="topics" default="['/image', '/point_cloud2']"/>
    <arg name="storage_path" default="/seerep/seerep-data/"/>
    <arg name="filename" default="3e2a9b3a-2ab0-48a7-87c2-7bf1ea6392a8"/>
    <arg name="project_root_frame" default="map"/>
    <arg name="project_name" default="testproject"/>
    <node name="seerep_ros_examples_hdf5_node" pkg="seerep_ros_examples" type="seerep_ros_examples_hdf5_node" output="screen">
        <rosparam param="topics" subst_value="True">$(arg topics)</rosparam>
        <rosparam param="storage_path" subst_value="True">$(arg storage_path)</rosparam>
        <rosparam param="filename" subst_value="True">$(arg filename)</rosparam>
        <rosparam param="project_root_frame" subst_value="True">$(arg project_root_frame)</rosparam>
        <rosparam param="project_name" subst_value="True">$(arg project_name)</rosparam>
    </node>
</launch>
